apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -506,14 +563,26 @@ lookup_from_local_node(BridgeType, BridgeName) ->
                         {503, error_msg('SERVICE_UNAVAILABLE', <<"request timeout">>)};
                     {error, Reason} ->
                         {500, error_msg('INTERNAL_ERROR', Reason)}
-                end;
+                end
+        end
+    ).
+
+'/bridges/:id/:operation'(post, #{
+    bindings :=
+        #{id := Id, operation := Op}
+}) ->
+    ?TRY_PARSE_ID(
+        Id,
+        case operation_func(Op) of
+            invalid ->
+                {400, error_msg('BAD_REQUEST', <<"invalid operation">>)};
```

Comment:
Q: isn't it also possible that the bridge does not exist on all nodes?

Reply:
Yes, I suppose so. We will return 500 on general RPC errors or 503 on RPC timeouts. Is there something specific you think we should handle? This part is functionally unchanged in the PR.

Reply:
I think that it was possible before that a [404 was returned by the operation](https://github.com/emqx/emqx/pull/9736/files#diff-9f021bc96805bb93b81edbbaa5c2b651147245181157ad75dba999129c1320f3L502).  Other than that, I don't recall any at the moment.  :thinking:

Reply:
That is true. But the 404 could only be returned by the enable/disable operation, and not by stop/restart. So the behavior is still consistent with the old API.